wazo_confd/plugins/extension_feature/service.py

```diff
@@ -1,22 +1,21 @@
-# Copyright 2017-2019 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2017-2023 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 from xivo_dao.helpers.db_manager import Session
-from xivo_dao.resources.extension import dao as extension_dao
+from xivo_dao.resources.feature_extension import dao as feature_extension_dao
 
 from wazo_confd.helpers.resource import CRUDService
 
 from .notifier import build_notifier
 from .validator import build_validator
 
 
-class ExtensionService(CRUDService):
+class FeatureExtensionService(CRUDService):
```

Comment:
I think search and get can be removed from this class

Reply:
@pc-m get/search removed